services/web-server/src/servers/createApp.js

```diff
@@ -19,7 +20,7 @@ module.exports = async ({ cfg, strategies, AuthorizationCode, AccessToken, auth,
 
   app.set('trust proxy', cfg.server.trustProxy);
   app.set('view engine', 'ejs');
-  app.set('views', 'src/views');
+  app.set('views', path.resolve(path.join(__dirname, '../views')));
```

Comment:
What's the difference in the output between

* `path.join(__dirname, '../views')`
* `path.resolve(path.join(__dirname, '../views'))`

I'm trying to see what is the benefit of adding path.resolve here other than just in case `../views` changes to an absolute path in the future.

Reply:
My understanding is that it changes `/app/services/web-server/src/servers/../views` to  `/app/services/web-server/src/views`, "resolving" the `..`.  I added it from a general sense that it's nicer to see paths without `..` in logs and things like that, but it's probably immaterial here.

Reply:
That makes a lot of sense. Thanks for elaborating.